docs/source/delta-sharing.md

```diff
@@ -0,0 +1,204 @@
+ 
+---
+description: Learn how to perform reads on Delta Sharing tables.
+---
+
+# Read Delta Sharing Tables
+
+Delta Sharing supports most of the options provided by <AS> DataFrame read for performing batch/streaming/cdf reads on tables.
```

Comment:
Given this is a first time we are adding this connector, it is good to add one or two sentences to introduce Sharing and point for further links.